src/main/java/org/apache/commons/csv/CSVFormat.java

```diff
@@ -846,6 +881,8 @@ public CSVFormat getFormat() {
     public static final CSVFormat EXCEL = DEFAULT.builder()

```

Comment:
I am wondering: Instead of this change, I wonder if we should have a new CSVFormat called `EXCEL_LENIENT`. Thoughts?

Reply:
Hello? Anyone?

Reply:
Maybe, but all spreadsheets are "lenient".

Reply:
>Maybe, but all spreadsheets are "lenient".
 
Good point. I think @garydgregory suggested the EXCEL_LENIENT format to avoid changing the behavior for users of the EXCEL format. But if we are changing the format to what's supported in Excel, I guess that should be fine.